app/config/Seeds/ArticlesSeed.php

```diff
@@ -0,0 +1,50 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Seed\AbstractSeed;
+
+/**
+ * Articles seed.
+ */
+class ArticlesSeed extends AbstractSeed
+{ 
+    public function getDependencies()
```

Comment:
```suggestion
    public function getDependencies(): array
```

Fatal error when run the command migrations seed as follow:

```
PHP Fatal error:  Declaration of ArticlesSeed::getDependencies() must be compatible with Phinx\Seed\AbstractSeed::getDependencies(): array in /app/config/Seeds/ArticlesSeed.php on line 11
```

When we override a method, we must follow the same declaration in parent class. And please ensure it's working as expected every time we make changes to the code.

Reply:
Thanks for the help `: array` it was the missing part to execute the command successfully.
Also added a queries to verify if the data already exist and used loggers if so.